infrastructure/nomad/playbooks/templates/jobs/contracts-deployer.nomad.j2

```diff
@@ -267,17 +267,35 @@ job "{{ job.name }}" {
           export SCRIPT_PATH_PREFIX="scripts"
           export CONTRACT_REPO_ROOT_PATH="./"
           LOGS="$(./entrypoint.sh)"
+          if [ $? -ne 0 ]; then
+            echo "Failed to deploy ${DEPLOY_TYPE} contracts!"
+            echo "${LOGS}"
+            exit 1
+          fi
+          end_time=$(date +%s)
+          echo "${DEPLOY_TYPE} contracts deployed successfully in: $(date -ud @$((end_time - start_time)) +'%H:%M:%S')."
+          echo "${LOGS}" \
+          | sed -n '/{.*}/p' \
+          | jq -c 'reduce .logs[] as $item ({}; . + {($item | split(": ")[0]): ($item | split(": ")[1])})' \
+          | jq -c 'with_entries(select(.key | startswith("_") | not))' \
+          > local/${DEPLOY_TYPE}_addresses.json
+
+          "${FORGE_BIN_PATH}" clean --root "${CONTRACT_REPO_ROOT_PATH}"
```

Comment:
This job is common for all deployments. We should restrict this only when we are deploying for testing. So there should be an if case here which will check if "beacon-emulator" is running. The beacon-emulator is the one returning these keys on the beacon API. So I think for now, we should restrict this to only when beacon-emulator is running.

Reply:
See https://github.com/primev/mev-commit/pull/615/commits/08f3c49a18cdb4b6955e09ea894484acf70ec3be